server/command.go

```diff
@@ -201,7 +201,7 @@ func (p *Plugin) runSendCommand(args []string, extra *model.CommandArgs) (bool,
 
 	receiverMessage := fmt.Sprintf("You have received a new Todo from @%s", senderName)
 
-	p.PostBotCustomDM(receiver.Id, receiverMessage, message, receiverIssueID)
+	p.PostBotCustomDM(receiver.Id, receiverMessage, message, "", receiverIssueID)
```

Comment:
The empty string pattern is a bit cryptic. Can we instead have a named variable we're passing in here, that is just assigned to an empty string? Same for other functions changed in this PR